Navigation/Profile/ProfileViewController.swift

```diff
@@ -7,48 +7,145 @@
+        
+        /// Register a cell. Makes a cell visible.
+        tableView.register(PostTableViewCell.self, forCellReuseIdentifier: String(describing: PostTableViewCell.self))
+        tableView.dataSource = self
+        tableView.delegate = self
+        /// Delete scroll indicator from right side.
+        tableView.showsVerticalScrollIndicator = true
+        return tableView
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        setUpProfileView()
+        view.addSubview(profileHeaderView)
+        
+        setUpProfileViewController()
     }
     
-    func setUpProfileView() {
-        view.backgroundColor = UIColor.lightGray
+    private func setUpProfileViewController() {
+        [tableView].forEach {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+            view.addSubview($0)
+        }
         
-        navigationController?.navigationBar.prefersLargeTitles = false
-        navigationItem.title = "Profile"
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            
+            profileHeaderView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            profileHeaderView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            profileHeaderView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            profileHeaderView.heightAnchor.constraint(equalToConstant: 220)
+        ])
+    }
+}
+
+extension ProfileViewContoller: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return PublicationStorage.publications.count
+    }
+    /// Number Of Rows In Section
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return PublicationStorage.publications[section].count
+    }
+    /// Shows row at section
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: String(describing: PostTableViewCell.self), for: indexPath) as? PostTableViewCell {
+            /// Преобразование ячейки. Наполняем ячеку. Обращение к двухмерному массиву.
+            cell.post = PublicationStorage.publications[indexPath.section][indexPath.row]
+            
+            return cell
+        }
         
-        view.addSubview(profileHeaderView)
-        view.addSubview(button)
+        return UITableViewCell()
+    }
+// MARK: Header for section.
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let profileHeaderView = ProfileHeaderView()
+
+        return profileHeaderView
```

Comment:
I would make a check for number of section, to make sure this header added to the first one

return section == 0 ? profileHeaderView : nil

Reply:
Thanks!